SRVCOM-1410: Adding serverless metrics (#33760)
openshift/openshift-docs: modules/serverless-event-source-metrics.adoc

[id="serverless-event-source-metrics_{context}"]
= Event source metrics

> You can use the following metrics to verify that events have been delivered from the event source to the connected event sink.

Reviewer: ```suggestion
You can use the following metrics to verify the delivery of events from the event source to the connected event sink.
```

Simplification purposes. Ignore if it changes the meaning of the sentence.

Author: I honestly prefer it as is, I think this suggestion would make it a little harder to understand
